src/flask/app.py

```
@@ -248,6 +248,8 @@ def __init__(
             instance_relative_config=instance_relative_config,
             root_path=root_path,
         )
+        # Add this in the Flask class __init__ method
+        self.secret_key = "hardcoded-secret-key-123"  # Security violation: Hardcoded secret key
```

Ruff (0.11.9)

Line too long (96 > 88)

(E501)